coord/src/module.c

```diff
@@ -2386,6 +2366,11 @@ static int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int
   MR_Init(cl, clusterConfig.timeoutMS);
   InitGlobalSearchCluster(clusterConfig.numPartitions, slotTable, tableSize);
 
+  if (clusterConfig.type == ClusterType_RedisOSS) {
+    // Initiate the topology updater after the MR cluster is initialized
+    InitRedisTopologyUpdater(ctx);
+  }
```

Comment:
This will cause setting auto memory on this ctx. I believe this did not happened before, we need to make sure it has no unwanted side effects.